dev-haskell/svg-builder/svg-builder-0.1.1-r1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
Usually when you do dependency-loosening version bumps I suggest just renaming ebuilds instead of copying those. That way it's easier to review actual changes (and chance of a revert is quite small).

Reply:
>  I suggest just renaming ebuilds instead of copying those

I'm afraid I don't understand. The original ebuild was `svg-builder-0.1.1.ebuild` and I named the new one `svg-builder-0.1.1-r1.ebuild` due to the `RDEPEND` changes. Specifically, this is the process I followed:

```sh
cd ${gentoo_haskell}/dev-haskell/svg-builder
hackport merge svg-builder # pulls in the new upperbounds from hackage
git diff # what changed? since rdeps changed...

mv svg-builder-0.1.1.ebuild svg-builder-0.1.1-r1.ebuild
# need to recover the original file
git checkout HEAD~ -- svg-builder-0.1.1.ebuild

# finally, all the end-matter
pkgcheck scan # any issues? fix and commit
repoman full -dx # same
git add .
git commit -s -m "dev-haskell/.....you-get-the-point" # I know I could use repoman commit here
```

Can you elaborate a bit on what you mean by "just renaming ebuilds"?

Reply:
`git checkout HEAD~ -- svg-builder-0.1.1.ebuild` restores original file. I think we can safely remove it as part of this change.

Reply:
Ah hah, ok I will remove the files whenever I do the `-rN` bump